src/Recurrence.ts

```diff
@@ -80,6 +80,8 @@ export class Recurrence {
                     referenceDate = window.moment(scheduledDate);
                 } else if (startDate) {
                     referenceDate = window.moment(startDate);
+                } else {
+                    referenceDate = window.moment();
                 }
 
                 if (!baseOnToday && referenceDate !== null) {
```

Comment:
`referenceDate` can no longer be `null`, right?

```suggestion
                if (!baseOnToday) {
```

Also, line 75 should now read `let referenceDate: Moment;` right?

Reply:
Happy to be helpful I really appreciate this plugin. I've made the changes to remove nullability from reference date so that all recurring tasks default to checking today's date when completed.